src/components/Tokens/TokenDetails/PriceChart.tsx

```diff
@@ -185,6 +156,54 @@ export function PriceChart({ width, height, token }: PriceChartProps) {
   // y scale
   const rdScale = scaleLinear().domain(getPriceBounds(pricePoints)).range([graphInnerHeight, 0])
 
+  function tickFormat(
+    startTimestamp: number,
+    endTimestamp: number,
+    timePeriod: TimePeriod,
+    locale: string
+  ): [TickFormatter<NumberValue>, (v: number) => string, NumberValue[]] {
+    const startDate = new Date(startingPrice.timestamp.valueOf() * 1000)
+    const endDate = new Date(endingPrice.timestamp.valueOf() * 1000)
+    switch (timePeriod) {
+      case TimePeriod.HOUR:
+        return [
+          hourFormatter(locale),
+          dayHourFormatter(locale),
+          timeMinute.range(startDate, endDate, 10).map((x) => x.valueOf() / 1000),
```

Comment:
how are you picking these constants? (10, 4, 1 etc)

Reply:
based on Figma: 
- for 1H chart ticks are 10m apart
- for 1D chart ticks are 4 hours apart
- for 1W chart ticks are 1 day apart

still have some adjustments to make for 1Y and All still